### PR TITLE
feat: return permission alongside item

### PR DESCRIPTION
**Pull request by pyphilia**
Would be nice to have a discussion on naming.
For now I've only added the permission to `get` and `getMany` but the idea would be to also have it for all items' GET endpoints.

### CODE REVIEW

**Comment by pyphilia** [comment]
ref #837

File: src/services/item/ItemWrapper.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  | 'description'
+  | 'extra'
+  | 'createdAt'
+  | 'creator'
+  | 'updatedAt'
+  | 'settings'
+  | 'lang'
+>;
+
+// TODO: export in sdk??
```
Comment (spaenleh) [inline src/services/item/ItemWrapper.ts:22]: So ? if we took a decision, I think we can remove this.

File: src/services/item/fluent-schema.ts [hunk 2]
```diff
@@ -327,5 +346,29 @@ export default {
...
+          additionalProperties: true,
+        },
+        settings: {
+          type: 'object',
+          additionalProperties: true,
+        },
+        creator: { $ref: 'https://graasp.org/members/#/definitions/member' },
+        createdAt: { type: 'string' },
+        updatedAt: { type: 'string' },
+        permission: { type: ['string', 'null'] },
```
Comment (spaenleh) [inline src/services/item/fluent-schema.ts:370]: set as enum ?

File: src/services/item/plugins/geolocation/index.test.ts [hunk 3]
```diff
@@ -522,7 +596,25 @@ describe('Item Geolocation', () => {
         ({ app, actor } = await build());
       });
 
-      it('get adress from search', async () => {
+      it.only('get address from search', async () => {
```
Comment (spaenleh) [inline src/services/item/plugins/geolocation/index.test.ts:599]: Remove `.only` ?

**Review by spaenleh — APPROVED** [review]
Wow ! looks good, just a few things to cleanup and we are good !

**Pull request merged**